ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-input-subtext/advanced-gas-fee-input-subtext.js

```diff
@@ -1,42 +1,71 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 
+import { isNullish } from '../../../../helpers/utils/util';
+import { formatGasFeeOrFeeRange } from '../../../../helpers/utils/gas';
+import { I18nContext } from '../../../../contexts/i18n';
 import Box from '../../../ui/box';
-import I18nValue from '../../../ui/i18n-value';
 import LoadingHeartBeat from '../../../ui/loading-heartbeat';
 
-const AdvancedGasFeeInputSubtext = ({ latest, historical, feeTrend }) => {
+const FEE_TRENDS = ['up', 'down', 'level'];
+
+const AdvancedGasFeeInputSubtext = ({ latest, historical, trend }) => {
+  const t = useContext(I18nContext);
   return (
-    <Box className="advanced-gas-fee-input-subtext">
-      <Box display="flex" alignItems="center">
-        <span className="advanced-gas-fee-input-subtext__label">
-          <I18nValue messageKey="currentTitle" />
-        </span>
-        <span className="advanced-gas-fee-input-subtext__value">
-          <LoadingHeartBeat />
-          {latest}
-        </span>
-        <span className={`advanced-gas-fee-input-subtext__${feeTrend}`}>
-          <img src={`./images/${feeTrend}-arrow.svg`} alt="feeTrend-arrow" />
-        </span>
-      </Box>
-      <Box>
-        <span className="advanced-gas-fee-input-subtext__label">
-          <I18nValue messageKey="twelveHrTitle" />
-        </span>
-        <span className="advanced-gas-fee-input-subtext__value">
-          <LoadingHeartBeat />
-          {historical}
-        </span>
-      </Box>
+    <Box
+      display="flex"
+      alignItems="center"
+      gap={4}
+      className="advanced-gas-fee-input-subtext"
+    >
+      {isNullish(latest) ? null : (
+        <Box display="flex" alignItems="center" data-testid="latest">
+          <span className="advanced-gas-fee-input-subtext__label">
+            {t('currentTitle')}
+          </span>
+          <span className="advanced-gas-fee-input-subtext__value">
+            <LoadingHeartBeat />
+            {formatGasFeeOrFeeRange(latest)}
+          </span>
+          {FEE_TRENDS.includes(trend) ? (
+            <span className={`advanced-gas-fee-input-subtext__${trend}`}>
+              <img
+                src={`./images/${trend}-arrow.svg`}
+                alt={`${trend} arrow`}
```

Comment:
Ah, you're right on this. I think this is here so that we can target the image in the test using `queryByAltText`. So I'm guessing that we should probably translate this, but what do you think?

Reply:
I've added this alt text to translations.